src/main/groovy/life/qbic/datamodel/dtos/portal/PortalUser.groovy

```diff
@@ -0,0 +1,46 @@
+package life.qbic.datamodel.dtos.portal
+
+import groovy.transform.Immutable
+
+/**
+ * <p>A user within QBiC's data management platform.</p>
+ *
+ * <p>For example a user with the full name <strong>Max Maxwell Minestrone</strong> has
+ * the first name <i>Max</i>, the middle name <i>Maxwell</i> and the last name <i>Minestrone</i>.</p>
+ *
+ * @since 2.9.0
+ */
+@Immutable
+class PortalUser {
+
+    /**
+     * The portal system's user id
+     * @since 2.9.0
+     */
+    String id
+
+    /**
+     * The full name of the user
+     * @since 2.9.0
+     */
+    String fullName
+
+    /**
+     * The first name of the user
+     * @since 2.9.0
+     */
+    String firstName
+
+    /**
+     * The middle name of the user
+     * @since 2.9.0
+     */
+    String middleName
```

Comment:
what happens if someone does not have a middle name?

Reply:
isnt that Optional?

Reply:
sure, then you just pass an empty string upon construction

Reply:
I would think it would mostly be like we provide the empty string which I personally dont like so much. In how many cases do we really have middle name?